kernel/main.cpp

```diff
@@ -61,31 +55,29 @@ static void boot_init(void *boot_info, uint32_t magic)
  * @brief This is the Panix kernel entry point. This function is called directly from the
  * assembly written in boot.S located in arch/i386/boot.S.
  */
-void kernel_main(void *boot_info, uint32_t magic) {
+void kernel_main(void* boot_info, uint32_t magic)
+{
     // Print the splash screen to show we've booted into the kernel properly.
     kernel_print_splash();
     // Install the GDT
     interrupts_disable();
-    gdt_install();                  // Initialize the Global Descriptor Table
-    isr_install();                  // Initialize Interrupt Service Requests
-    rs232::init(RS_232_COM1);        // RS232 Serial
-    paging_init(0);                 // Initialize paging service (0 is placeholder)
-    boot_init(boot_info, magic);    // Initialize bootloader information
-                                    // TODO: Bootloader should be first but currently
-                                    //       requires paging, which should come after
-                                    //       boot information is parsed.
+    gdt_install();               // Initialize the Global Descriptor Table
+    isr_install();               // Initialize Interrupt Service Requests
+    rs232::init(RS_232_COM1);    // RS232 Serial
+    boot_init(boot_info, magic); // Initialize bootloader information
+    paging_init(0);              // Initialize paging service (0 is placeholder)
     fb::init(handoff.getFramebufferInfo());
-    kbd_init();                     // Initialize PS/2 Keyboard
-    rtc_init();                     // Initialize Real Time Clock
-    timer_init(1000);               // Programmable Interrupt Timer (1ms)
+    kbd_init();       // Initialize PS/2 Keyboard
```

Comment:
Once again, I know this isn't your fault, but could you re-align these comments?

Reply:
Fixed em.